account_asset_management/account_asset.py

```diff
@@ -605,6 +605,20 @@ def _get_depreciation_entry_name(self, cr, uid, asset, seq, context=None):
         return (asset.code or str(asset.id)) + '/' + str(seq)
 
     def compute_depreciation_board(self, cr, uid, ids, context=None):
+        """
+        Computes and stores depreciation lines for the given asset ids.
+
+        CAUTION: make sure the last depreciation line with init_entry=True
+        contains only depreciated amount for its respective fiscal year or this
+        method may compute wrong depreciation lines.
```

Comment:
This is also possible to complete the help on the field "Asset Start Date" in the view ?